1906-10-08.xml

```diff
@@ -1353,9 +1349,1022 @@
+                    during that time 49 vessels transited the Canal, 21 being northward, and 28 soutward bound.</p></div>
+                                <div type="item"><p>Swimming Club Sport.</p>
+                    <p>A full account of the fourth annal aquatic sport of the Alexandria Swimming Club, which 
+                    were held on Saturday afternoon, will be found on page 5. An excellent caristure of what we
+                    might call the "clowns' parade." prevision to walking the pole in costume, was
+                    drawn by Mr. T.A. Binder, and is on a view in the window of Messers' Baiser and Binder, 
+                    photographers.</p></div>
+                                    <div type="item"><p>The Plague.</p>
+                    <p> During the week ended on Saturday ten cases in all were notified in Egypt, seven
+                    of which occured at the Suez, and three at Alexandria. SEven deaths are reported and seven 
+                    cases were discharged. The number of cases under treatment on that day was eight. Since the 
+                    1st of January there have been in all 486 cases, against 260 and 812 during the corresponding
+                    periods of 1905 and 1904 respectively. Since Saturday no cases have been reported.</p></div>
+                                        <div type="item"><p>All Saint's Church.</p>
+                    <p>There will be an All Saints' Jamble Sale again this year; it will probaly be held about the
+                    usual date, that is, early in November. There are sundry  charities and expenses connected 
+                    with the Church for which a fund 1 k this is exceedingly useful. Last year the sale realized 
+                    over £30. So if there is any kind of aritcle not required, Cairo Anglicans should remember that
+                    it will be most acceptable for the jamble sale, even though it should seem to be no value at all.
+                    Any thing for sale may be sent to Church House.</p></div>
+                                            <div type="item"><p>Cambodian Heriosts for Canal Omoiais.</p>
+                    <p>King Sisowath of Camboida says our confere that "Verite" of Port Said, conferred the Royal
+                    Order of Merit of Camboida on M. Con'lant, princple agent of the transit department
+                    at Ismailieh, M. Domont, principal agent of the same service at Port Said, M. Bougeret, 
+                    deput principal agent at Suez and M. Viemont, chief clerk of the Canal Telegraph Servic. 
+                    These decorations are conferrred in recogniition of services rendered to his Majesty during his passage
+                    through the Canal in August on his return to Cambodia from France.</p></div>
+                </div>
+                <cb n="2"/>              
+                <div type="item">
```

Comment:
it looks like this whole column is a section of stories from the newswire. These all have a dateline and say Reuter or Havas at the end. You could wrap the whole column in a `<div type="section" element="wire>` tag.